src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,179 @@
+import axios from 'axios'
+import shared from 'botpress/shared'
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+import { EventData, EventPackageInfoType, StoreInfoType, TelemetryPackage } from './telemetry_type'
+
+export const telemetryPackageVersion = '1.0.0'
+export const dataClusterVersion = '1.0.0'
+
+export const endpoint = 'https://telemetry.botpress.dev'
+
+export const corsConfig = {
+  withCredentials: false
+}
+
+export const storeInfos: StoreInfoType = {}
+
+export const eventPackageInfo: EventPackageInfoType = {}
+
+export function toHash(content: string) {
+  return createHash('sha256')
+    .update(content)
+    .digest('hex')
+}
+
+export function switchLock(lockKey: string) {
+  if (_.has(eventPackageInfo, lockKey)) {
+    eventPackageInfo[lockKey].locked = !eventPackageInfo[lockKey].locked
+  }
+}
+
+function isTimeoutLocalStorage(event: string) {
+  const item = window.localStorage.getItem(event)
+  if (item !== null) {
+    const timeout = parseInt(item) - new Date().getTime()
+    if (timeout >= 0) {
+      return true
+    }
+    window.localStorage.removeItem(event)
+  }
+  return false
+}
+
+function getTimeoutLocalStorage(event: string) {
+  const item = window.localStorage.getItem(event)
+  if (item !== null) {
+    const timeout = parseInt(item) - new Date().getTime()
+    if (timeout > 0) {
+      return timeout
+    }
+    window.localStorage.removeItem(event)
+  }
+  return 0
+}
```

Comment:
Duplicates. Duplicates everywhere. 
Exports, Exports everywhere

Reply:
Yeah... forgot to remove the old telemetry.ts

Reply:
Usually when you open a pull request, the best thing to do is a self-review before asking others to review it. Saves a lot of time